apps/web/src/pages/Pools/PoolAdvertisementPage/components/LanguageRequirementDialog.tsx

```diff
@@ -0,0 +1,461 @@
+            <Heading
+              level={headingLevel}
+              size="h6"
+              data-h2-margin-top="base(0)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Bilingual positions",
+                id: "i9ovz7",
+                description: "Sub-heading for language requirements dialog",
+              })}
+            </Heading>
+            <div data-h2-display="base(grid)" data-h2-gap="base(x.5)">
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "You do not need to have your second language proficiency levels before applying for a bilingual position. You will be sent for testing as part of the hiring process.",
+                  id: "++a1ZY",
+                  description:
+                    "Description for bilingual positions in language requirements dialog",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage({
+                  defaultMessage:
+                    "If you are applying for a bilingual position, you must pass the language tests before you can be hired. If the position is bilingual, the manager cannot hire someone who does not meet the second language proficiency levels, no matter what other qualifications the candidate may have or how well they perform in the job competition. If you are interested in a bilingual job but are not bilingual, you are strongly encouraged to apply to other opportunities on the platform that meet your language abilities.",
+                  id: "53MCSe",
+                  description:
+                    "Description for bilingual positions in language requirements dialog",
+                })}
+              </p>
+              <p>
+                {intl.formatMessage(
+                  {
+                    defaultMessage:
+                      "Try out the <link>self-assessment</link> tests to get a sense of your current level.",
+                    id: "x20cWU",
+                    description:
+                      "Description for bilingual positions in language requirements dialog",
+                  },
+                  {
+                    link: (chunks: ReactNode) =>
+                      selfAssessmentLink(locale, chunks),
+                  },
+                )}
+              </p>
+            </div>
+          </div>
+          <div>
+            <Heading
+              level={headingLevel}
+              size="h6"
+              data-h2-margin-top="base(0)"
+            >
+              {intl.formatMessage({
+                defaultMessage: "Second language proficiency",
+                id: "fSeYdG",
+                description: "Sub-heading for language requirements dialog",
+              })}
+            </Heading>
+            <div
+              data-h2-display="base(grid)"
+              data-h2-gap="base(x.5)"
+              data-h2-margin-bottom="base(x.5)"
+            >
```

Comment:
This seems to be causing some inconsistent spacing. I'm not sure it is entirely necessary either if we rely on the margin of each element.

Reply:
Fixed by [7957bec](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10628/commits/7957bec46d036ea078394010540617660e66b9df)